source/chapters/user_interface.rst

```diff
@@ -666,15 +676,28 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
-  Above these knobs, the gain knob allows you to adjust the gain of the deck. In
-  general, you should adjust this knob so that at full-volume the deck's audio
-  is just at the peak of the center VU meters. This is so you can achieve the
-  widest dynamic range for your track.
+  Above these knobs, the gain knob allows you to adjust the gain of the deck. 
+  Use this to compensate for the differences in recording levels between tracks.
+  In general, you should adjust this knob so that at full-volume the deck's
+  audio is just at the peak of the center VU meters. This is so you can achieve
+  the widest dynamic range for your track.
+  
+  .. note:: By default, Mixxx automatically adjusts the gain of tracks to sound
+            like they are at approximately the same volume. Your tracks must be
+            :ref:`analyzed<library-analyze>` in advance to take advantage of
+            this feature.
 
 **EQ Knobs**
   The low, mid, and high knobs allow you to change the filters of the audio.
   This allows you to selectively reduce or boost certain frequency ranges of
   the audio.
+  
+  .. warning:: Be careful not to raise the EQs so much that any of the VU meters
+               are going into the red. At that point, your audio is clipping and
+               will sound worse. This is more often an issue with the low EQ
+               than the mid or high EQs. If you want to emphasize a frequency
+               band, it is better to lower the other ones and boost the track's
+               overall gain instead.
```

Comment:
This is the use for the yellow region. .. for momentary band emphasize. 
If you need a permanent band boost, you should lower the gain or the target loudness in normalization preferences in a way that the yellow region is not used.